uri-schemes.md

```diff
@@ -124,6 +126,88 @@ HTTP Status code: NOT 2XX, idealy the 4XX if it's user error and 5XX if it's the
+| ---------- | ------------------------- | --------------------------------------------------- | ----------------      |
+| `ih`       | `imap_host`               | IMAP host                                           | `ih=imap.example.com` |
+| `ip`       | `imap_port`               | IMAP port                                           | `ip=993`              |
+| `iu`       | `imap_username`           | IMAP username                                       |                       |
+| `ipw`      | `imap_password`           | IMAP password                                       |                       |
+| `is`       | `imap_security`           | IMAP security: "`ssl`" or "`default`"               | `is=ssl`              |
+| `ic`       | `imap_certificate_checks` | IMAP certificate checks, see below for options      | `ic=1`                |
+| ---------- | ------------------------- | --------------------------------------------------- | ----------------      |
+| `sh`       | `smtp_host`               | SMTP host                                           | `sh=mail.example.com` |
+| `sp`       | `smtp_port`               | SMTP port                                           | `sp=465`              |
+| `su`       | `smtp_username`           | SMTP username                                       |                       |
+| `spw`      | `smtp_password`           | SMTP password                                       |                       |
+| `ss`       | `smtp_security`           | SMTP security: "`ssl`", "`starttls`" or "`plain`"   | `ss=plain`            |
+| `sc`       | `smtp_certificate_checks` | SMTP certificate checks, see below for options      | `sc=3`                |
+
+#### `minVersion`
+
+Format version:
+Used for breaking new versions that add new **required** properties, basically deltachat checks for this and if it's newer than the version deltachat supports it prompts the user to update the app.
+
+The version number only increases on incompatible changes (changes to required properties).
+
+#### `CertificateChecks`
+
+| code | name & description                                                                                               |
+| ---- | ---------------------------------------------------------------------------------------------------------------- |
+| 0    | `Automatic`, same as `AcceptInvalidCertificates` unless overridden by `strict_tls` setting in provider database. |
+| 1    | `Strict`                                                                                                         |
+| 3    | `AcceptInvalidCertificates`                                                                                      |
+
+#### Important information for using the `DCLOGIN:` scheme
+
+- There is a maximum length of how much data fits in side of a qr code (depending on the error correction level 1273 chars to 2953 chars)
+  - make sure to use the short names for advanced properties
```

Comment:
> maybe even bugs when code paths are used rarely.

probably a match statement or sth similar, so it's not that likely it would produce bugs. 
But right we don't need to implement parsing if we don't want it to be used in the first place.